bundles/org.openhab.binding.mideaac/README.md

```diff
@@ -0,0 +1,76 @@
+## Midea AC Binding
+
+This binding integrates Air Conditioners that use the Midea protocol. Midea is an OEM for many brands.
+
+An AC device is likely supported if it uses one of the following Android apps or it's iOS equivalent.
+ 
+| Application						 		 |				Comment									 |
```

Comment:
Please align this and the other tabels with some markdown tool or markdowntable.com

Reply:
Done